src/checks/__main__.py

```diff
@@ -91,12 +92,16 @@ def main() -> None:
             except DataSourceError:
                 gdf = GeoDataFrame()
             gdfs.append(gdf)
-        for function, results in checks:
-            rows = function.main(iso3, gdfs)
-            results.extend(rows)
+        with Pool() as pool:
```

Comment:
Nice, I think this is the right approach, as most of the runtime seems to be CPU bound.